tests/test_redshift/test_redshift.py

```diff
@@ -339,7 +339,7 @@ def test_create_cluster_with_vpc_security_groups_boto3():
 
 @mock_redshift
 def test_create_cluster_with_iam_roles():
-    iam_roles_arn = ['arn:aws:iam:::role/my-iam-role',]
+    iam_roles_arn = ['arn:aws:iam:::role/my-iam-role', ]
```

Comment:
Why add a space when you could remove the `,`

Reply:
Think pycharms or something was being silly. Need me to update it?

Reply:
Nah it'll do for now, was merged yesterday